compliance_checker/tests/test_ioos_profile.py

```diff
@@ -291,3 +294,217 @@ def test_good_units(self):
         results = self.ioos.check_units(self.ds)
         for result in results:
             self.assert_result_is_good(result)
+
+class TestIOOS1_2(BaseTestCase):
+    '''
+    Tests for the compliance checker implementation of IOOS Metadata Profile
+    for NetCDF, Version 1.1
+    '''
+
+    # for reference
```

Comment:
Is this showing what the code is equivalently generating underneath the hood?

Reply:
Nope, good catch - that was purely an inline comment when developing. I will remove.